snippets/loader-component.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Loader Component
+level: beginner
+author: thekiba
+tags:
+  - tips
+  - good-to-know
+  - components
+  - templates
+---
+
+# Content
+You can create own helper component and use it instead of `*ngIf`.
+
+```typescript
+@Component({
+  selector: 'loader',
+  template: `
+    <ng-content *ngIf="!loading else showLoader"></ng-content>
+    <ng-template #showLoader>
+      Come away with me to the Anywhere City!
+    </ng-template>
+  `
+})
+class LoaderComponent {
+  @Input() loading: boolean;
+}
+```
+
+For usage example:
+```html
+<loader [loading]="loading">🦊 🦄 🐉</loader>
```

Comment:
Shouldn't this be a * directive?

Reply:
It's should be component. But it can be a * directive too.